evap/staff/importers.py

```diff
@@ -97,28 +105,111 @@ def store_in_database(self, vote_start_datetime, vote_end_date, semester):
         assert not self.errors
         # This is safe because the user's email address is checked before in the importer (see #953)
         responsible_dbobj = UserProfile.objects.get(email=self.responsible_email)
-        course = Course(
-            name_de=self.name_de,
-            name_en=self.name_en,
-            type=self.course_type,
+        if self.existing_course:
+            self.existing_course.degrees.add(*self.degrees)
+        else:
+            course = Course(
```

Comment:
If you would change it to this:
```python
        if self.existing_course:
            self.existing_course.degrees.add(*self.degrees)
            return
        course = Course(
        # ...
```

you wouldn't have the large diff for the else-branch